README.md

```diff
@@ -25,17 +25,66 @@ What this tool does:
 Just open up your `Terminal.app` on OS X, copy-paste this into it and hit Enter
 to run:
 
-For `Xcode` project (project or workspace):
+#### Xcode
+<details><summary>For Archiving & Exporting IPA for `Xcode` project (project or workspace):</summary>
```

Comment:
These **`** s are not working as expected.
![Screen Shot 2019-03-18 at 10 56 25 AM](https://user-images.githubusercontent.com/1564933/54522131-c318f880-496c-11e9-8b93-490c2b7f8f10.png)

Reply:
Yeah, markdown embedded in HTML tags is a no-go, I beleive. :(

Reply:
Fixed in:
https://github.com/bitrise-tools/codesigndoc/pull/99/commits/e82846a30386b287d32208390cfabb329bd42bcd